ui/desktop/README.md

```diff
@@ -31,32 +31,42 @@ You will need the following things properly installed on your computer.
 
 ## Installation
 
-See boundary-ui README for installation instructions.
+See [boundary-ui README](https://github.com/hashicorp/boundary-ui#installation) for installation instructions.
```

Comment:
Just adding a link to the Boundary-UI readme. But if you think this project is getting mature enough to describe its own installation process happy to discuss.